pkg/commands/compute/compute_unit_test.go

```diff
@@ -183,6 +159,32 @@ func TestFileNameWithoutExtension(t *testing.T) {
 }
 
 func TestGetIgnoredFiles(t *testing.T) {
+	// we're going to chdir to a build environment,
```

Comment:
The changes to this function can be summarised as follows...

1. Define a single temp environment rather than having a new environment for each test scenario.
2. For scenario specific files, create those within the temp environment separately.